src/OrchardCore.Modules/OrchardCore.Title/Views/TitlePartSettings.Edit.cshtml

```diff
@@ -21,6 +21,14 @@
         </select>
     </div>
 </div>
+<div class="form-group pb-1">
+    <div class="custom-control custom-checkbox">
+        <input asp-for="RenderTitle" class="custom-control-input" />
+        <label asp-for="RenderTitle" class="custom-control-label">@T["Render Title"]</label>
+        <span class="hint">@T["Check to renter the title in views.)"]</span>
```

Comment:
```suggestion
        <span class="hint">@T["Check to render the title in views.)"]</span>
```

Did you disable the permssion for us to update your PRs directly?

Reply:
No I didn’t change anything